src/js/prop-types/prop-types.js

```diff
@@ -0,0 +1,21 @@
+import PropTypes from 'prop-types';
+
+function getShim(propType) {
+  function shim(...args) {
+    return propType(...args);
+  }
+
+  Object.keys(propType).forEach((key) => {
```

Comment:
I did this extra loop mainly for `isRequired` support.
Though I may forgot to shim these too 🤔